### ISSUE TITLE
Rename static files that are templates

### ISSUE THREAD

**Issue by gedw99** [golang-cafe/job-board#101]
In https://github.com/golang-cafe/job-board/tree/master/static/views it’s hard to see what’s a reused template and what’s a complete html page.

Can we just rename all template ones to start with a prefix if “tpl_” ?

the refactor is a simple find and replace from what I can see .

This will make it alot easier to work on and extend and is not disruptive to the rest of the architecture

**Comment by dm161** [comment]
@gedw99 that's a good idea, when i implemented templates i was thinking to move them into a different subdirectory but we might need some more refactoring around how we load templates in Go. I can allocate some budget here to move them into subdirectory. views/**.html are views and templates/**.html are templates. For templates there is a lot of potential improvements, like splitting common css/js/html into re-usable templates, right now there is a lot of copy pasting going on. I can create new tickets for these also.

**items**
- creates distinct directories for views and templates
- update golang code accordingly so that templates and views are processed correctly

/bounty $10

**Comment by gedw99** [comment]
Agreed.

but I would not create sun folder but instead use file name prefixes. It’s a sneaky form of tagging 

why though ?

because file hierarchies make templates more complex with subtle bugs. 

I feel it’s yagni! The pkg folder is nicely one folder per service and it’s why it’s so easy to grok the system.

**Comment by sareensumanau** [comment]
I agree with @gedw99 , in case we have to create folder, we should think of it as a module wise, for eg: keeping all html and templates of let's say developer module in a folder of its own.

As v1, we should at-least attach prefix to all templates html.

**Comment by jemiluv8** [comment]
I'm working on this.

**Comment by jemiluv8** [comment]
There were only three template files and they were all suffixed with -template.
So perhaps I thought there was some sort of "indicator" already as to whether it was
full page or reused template. 

I also thought "tpl-" might be more consistent with the file naming convention in the static/views folder
(instead of tpl_).

But I went ahead with the changes as discussed. Awaiting feedback ...

**Comment by dm161** [comment]
we already have -template suffix, if we have to do any change at all to the templating structure it will be subfolders, otherwise there is no point in investing time on this

**Comment by jemiluv8** [comment]
Yeah I agree. So do we decline the PR?
